src/applications/personalization/profile360/reducers/vet360.js

```diff
@@ -72,9 +72,9 @@ export default function vet360(state = initialState, action) {
         fieldTransactionMap: {
           ...state.fieldTransactionMap,
           [action.fieldName]: {
+            ...state.fieldTransactionMap[action.fieldName],
             isPending: false,
-            transactionId: action.transaction.data.attributes.transactionId,
-            ...state.fieldTransactionMap[action.fieldName]
+            transactionId: action.transaction.data.attributes.transactionId
           }
```

Comment:
The order here had to be switched so that `isPending` is properly overwritten

Reply:
we should add an explanatory comment for clarity in the next PR